dragonfly/implementations/multido.py

```diff
@@ -162,7 +168,7 @@ def on_set(self, value):
                 logger.debug('no target_value given: using value ({}) as a target_value to check'.format(value))
                 target_value = value
             if value_get==None:
```

Comment:
not a line you changed, but you should not check `==None`, should be `if value_get is None`

Reply:
good catch, that shows up a couple times, now fixed